src/app/file-browser/components/edit-tags/edit-tags.component.spec.ts

```diff
@@ -198,21 +198,6 @@ describe('EditTagsComponent', () => {
     expect(tagCreateSpy).not.toHaveBeenCalled();
   });
 
-  it('should open dialog when manage link is clicked', async () => {
-    const { element, find, inject, fixture } = await defaultRender();
-    const dialogOpenSpy = inject(Dialog);
-
-    element.componentInstance.isEditing = true;
-    fixture.detectChanges();
-
-    find('.manage-tags-message .manage-tags-link').triggerEventHandler(
-      'click',
-      {},
-    );
-
-    expect(dialogOpenSpy.open).toHaveBeenCalled();
-  });
-
```

Comment:
I think this is an example of a deleted test that didn't need to be fully deleted. If I comment out or remove the dialog call completely, tests will still pass. Do we really want this behavior to not be covered by tests?